nservicebus/sqlserver/concurrency_content_sqlserver_[,2).partial.md

```diff
@@ -0,0 +1,3 @@
+Each endpoint running SQL Server transport spins up a fixed number of threads (controlled by `MaximumConcurrencyLevel` property of `TransportConfig` section) both for each input queue. Each thread runs in a loop, polling the database for messages awaiting processing.
+
+As a consequence, if `MaximumConcurrencyLevel` is set to 10, there are 40 threads running and constantly polling the database (10 threads for the main queue, 10 for each of satellite).
```

Comment:
Satellite comes up out of the blue, with no introduction and no connection with the previous paragraph. 

How about:

By default, there are 4 input queues created for every endpoint: (list them, e.g. Timeouts queue,...). As a consequence, ....

> 10 for each of satellite

10 for each input queue